taskcluster/test-training+deepspeech-linux-amd64-py27-opt.yml

```diff
@@ -0,0 +1,14 @@
+build:
+  template_file: test-linux-opt-base.tyml
+  dependencies:
+    - "linux-amd64-ctc-opt"
+    - "linux-amd64-cpu-opt"
+  system_setup:
+    >
+      apt-get -qq -y install ${python.packages.apt}
+  args:
+    tests_cmdline: "${system.homedir.linux}/DeepSpeech/ds/tc-train-tests.sh 2.7.13 mozilla deepspeech"
+  python_wheel: 'https://index.taskcluster.net/v1/task/project.deepspeech.tensorflow.pip.master.cpu/artifacts/public/tensorflow_warpctc-1.4.0-cp27-cp27mu-linux_x86_64.whl'
+  metadata:
+    name: "DeepSpeech Linux AMD64 CPU training Py2.7 (DS)"
+    description: "Training a DeepSpeech LDC93S1 model for Linux/AMD64 Python 2.7 with DeepSpeech, CPU only, optimized version"
```

Comment:
Maybe "with DeepSpeech package installed", to be more clear on what we're testing.